### PR TITLE
Fix Build Issues

### PR DESCRIPTION
**Pull request by halotroop2288**
- Chocoloom was outdated
	- Fixed by updating it to the latest commit
- CoreLib was impossible to resolve, causing Gradle setup to fail
	- Fixed by using a local JAR
- There was a recursive dependency that prevented build
	- Fixed by replacing choco's solution of including subprojects with asie's solution from FabricMC.
- Gradle 4.9 is hella old and not worth maintaining.
	- Fixed by updating Gradle to 6.9 (final Java 8 release).

### CODE REVIEW

File: gradle.properties
Comment (valoeghese) [inline gradle.properties:11]: our package doesn't have the dashes

Comment (valoeghese) [reply gradle.properties:11]: therefore our maven group doesn't

Comment (valoeghese) [reply gradle.properties:11]: yes I know our website has them but I'm not refactoring every project on this org just for that

File: gradle.properties
Comment (valoeghese) [inline gradle.properties:10]: is this worth bumping ver for?

**Review by valoeghese — CHANGES_REQUESTED** [review]
not much to say. I think since this is more setup based the version doesn't really need to be updated since there's no actual code changes. Also Chocohead could provide a better review here, I'm not a gradle guru like him. I'll wait for choco's review before a merge

File: gradle.properties
Comment (valoeghese) [reply gradle.properties:11]: also this is what we're under on half's maven already it's weird to change it

Comment (halotroop2288) [reply gradle.properties:11]: > yes I know our website has them but I'm not refactoring every project on this org just for that

I don't expect you to. Dashes are invalid for package names.
But the Maven group *should* match the domain name.

The reason I changed it is because when I switched from the online copy to my local copy, the dependency wasn't found until I changed it accordingly.

File: gradle.properties
Comment (halotroop2288) [reply gradle.properties:10]: Beats me. I changed it because I modified the buildscript and made some minor changes to fix warnings (but I didn't push them because they were *so* minor.)

Also, I added a copy of CoreLib. I feel that also makes for a bit of reason.

File: gradle.properties
Comment (halotroop2288) [reply gradle.properties:11]: And it already has dashes in the example mod.

https://github.com/minecraft-cursed-legacy/Example-Mod/blob/b1.7.3/build.gradle#L62

> also this is what we're under on half's maven already it's weird to change it

Half's maven isn't browseable, so all I know is MCL isn't mentioned in the unformatted XML file it links to, and I can't resolve it as a dependency using that repo, so it must not be in there at all.

**Comment by halotroop2288** [comment]
OK even this doesn't work. Using file Gradle dependencies creates null Maven dependencies at publish, which are impossible to resolve. It took at LOT of testing to figure this out.

The only solution is to either merge CoreLib into Cursed Legacy API, OR publish CoreLib to Maven properly.

**Pull request closed without merging**
